internal/monitors/expvar/config.go

```diff
@@ -57,6 +58,8 @@ type MetricConfig struct {
 	Type string `yaml:"type" validate:"required,oneof=gauge cumulative"`
 	// Metric dimensions
 	DimensionConfigs []DimensionConfig `yaml:"dimensions"`
+	// Path separator character of metric value in JSON object
+	PathSeparator string `yaml:"pathSeparator" default:"/"`
```

Comment:
Is this default fully backwards compatible with what this monitor did before?

Reply:
It kind of seems like the old default was `.`.

Reply:
@keitwb Ok I'm made the old default character the default. But I have noticed that the default path separator character is not set if you configure the embedded MetricConfig and don't provide a value for pathSeparator!